test/functional/Jsr292/src/com/ibm/j9/jsr292/CustomClassLoader.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2001, 2018 IBM Corp. and others
+ * Copyright (c) 2001, 2020 IBM Corp. and others
```

Comment:
moving a file doesn't change it's copyright

Reply:
Fixed.